src/pages/journal/journal.ts

```diff
@@ -216,6 +217,12 @@ export class JournalPage extends BasePageComponent implements OnInit {
       (<SlotComponent>componentRef.instance).hasSlotChanged = slot.hasSlotChanged;
       (<SlotComponent>componentRef.instance).showLocation = (slot.slotData.testCentre.centreName !== lastLocation);
       lastLocation = slot.slotData.testCentre.centreName;
+
+      // if this is a personal commitment assign it to the component
+      if (slot.component.name === PersonalCommitmentSlotComponent.name) {
+        (<PersonalCommitmentSlotComponent>componentRef.instance).personalCommitments =
```

Comment:
Can we not assign this no matter the slot type as it's on the SlotComponent interface. It just won't be used on the other ones

Reply:
Yes a good point. I'll take a look now.